CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Unreleased Changes
 ------------------
 
+* Feature - Officially supports Ruby version 3.3.
```

Comment:
This changelog entry isn't necessary - It's already supported since we have minimum Ruby versions of the gem, but it's useful to have it in CI.